src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Commands/SetAzureRmAnalysisServicesServer.cs

```diff
@@ -88,6 +88,22 @@ public int ReadonlyReplicaCount
             HelpMessage = "Firewall configuration")]
         public PsAzureAnalysisServicesFirewallConfig FirewallConfig { get; set; }
 
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false,
```

Comment:
same comment

Reply:
@markcowl updated, please review again.

Reply:
These should be in separate parameter sets.

Reply:
@markcowl updated